assets/js/app.js

```diff
@@ -0,0 +1,9 @@
+$('.start').html(AppTemplates.start(heroes));
```

Comment:
I would reccomend changing this from `.start` to `.game` or you will have to hide and show different screens instead of just "switching" between start, battle, and end